plugins/base/src/main/kotlin/DokkaBase.kt

```diff
@@ -71,4 +73,28 @@ class DokkaBase : DokkaPlugin() {
     val fileWriter by extending(isFallback = true) {
         outputWriter providing ::FileWriter
     }
+
+    val rootCreator by extending {
+        htmlPreprocessors with RootCreator order { before(navigationPageInstaller) }
+    }
+
+    val navigationPageInstaller by extending {
+        htmlPreprocessors with NavigationPageInstaller order { before(searchPageInstaller) }
+    }
+
+    val searchPageInstaller by extending {
+        htmlPreprocessors with SearchPageInstaller order { before(resourceInstaller) }
+    }
+
+    val resourceInstaller by extending {
+        htmlPreprocessors with ResourceInstaller order { before(styleAndScriptsAppender) }
```

Comment:
Do we need the ordering here? Looks like those preprocessors are independent but I'm not sure

Reply:
bump